webpack.config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 const path = require('path');
```

Comment:
Here too, if we can we could use import instead of override rule

Reply:
I had a brief conversation about this with a pro coder and he said it would require "a lot of duct tape" 😄

Reply:
Maybe not worth it 😄

Reply:
Sounds legit